charts/camunda-platform/templates/_helpers.tpl

```diff
@@ -124,6 +124,14 @@ Set imagePullSecrets according the values of global, subchart, or empty.
     {{- end -}}
 {{- end -}}
 
+{{- define "camundaPlatform.keycloakBaseURL" -}}
```

Comment:
Let's call it `camundaPlatform.keycloakURLBase` and use it inside `camundaPlatform.keycloakURL`.